### ISSUE TITLE
Time for an update?

### ISSUE THREAD

**Issue by richardhendricks** [selfup/avorion#9]
Thanks for the great tool!  With Avorion 2.0 it looks like the devs have changed up a lot of the economics, ie mines need goods now too. Any plans for an update?

**Comment by selfup** [comment]
I will have to check it out! Thanks for the heads up. I will try to find time this weekend.

Currently it looks like this raw data may not be up to date: https://avorion.gamepedia.com/Goods?action=raw

**Comment by richardhendricks** [comment]
Yes unfortunately. I haven't looked into if there is an automated way to slurp in the data files. :(

**Comment by selfup** [comment]
I will give the wiki some time. If in a few months there is no change I'll give it a whirl!

**Comment by selfup** [comment]
I think I was able to add the changes you asked for. The wiki has been updated so I wrote a new parser

<img width="1069" height="587" alt="Image" src="https://github.com/user-attachments/assets/db6ddf6a-ca89-4345-9da3-90dcaf8b7e81" />

<img width="362" height="622" alt="Image" src="https://github.com/user-attachments/assets/866476cc-1c5f-4a99-9837-a4fced68cbea" />
